src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+const ADD = 'ADD';
```

Comment:
- [ ] [OPTIONAL] The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const ADD_BOOK = bookstore/books/ADD_BOOK` and `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`. This is essential if you have a large project, as you can easily differentiate the various actions that you have in the project. For example, you are using `const ADD = 'ADD';` which can be confusing for any other developer in your team. But if you use something like `const ADD = 'bookstore/books/ADD_BOOK';` this is more descriptive and helpful to both you and other team members.

Please fix this.

Reply:
Thank you @bobb-Rob for your helpful code review. I will definitely change action values to make it easier for others and myself to grasp.